src/facilities/hcf/views/pages/Prefroutes.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <div class="mt-10">
+    <h1 class="text-3xl text-white">Honolulu Control Facility Preferred Routing</h1>
+    <div class="search mt-10">
+      <div class="flex items-center space-x-4">
+        <div class="relative">
+          <label class="block text-sm font-medium text-white">
+            From:
+          </label>
+          <div class="mt-1 relative rounded-md shadow-sm">
+            <input
+              v-model="selectedFrom"
+              type="text"
+              class="input-box focus:ring-indigo-500 focus:border-indigo-500 pl-3 pr-4 text-black border-gray-300 rounded-md w-40 sm:w-48 h-12"
+              placeholder="Departure ICAO"
+            />
+          </div>
+        </div>
+        <div class="relative">
+          <label class="block text-sm font-medium text-white">
+            To:
+          </label>
+          <div class="mt-1 relative rounded-md shadow-sm">
+            <input
+              v-model="selectedTo"
+              type="text"
+              class="input-box focus:ring-indigo-500 focus:border-indigo-500 pl-3 pr-4 text-black border-gray-300 rounded-md w-40 sm:w-48 h-12"
+              placeholder="Arrival ICAO"
+            />
+          </div>
+        </div>
+
+        <button
+          @click="search"
+          class="button h-12 px-4 mt-6 border-2 border-white rounded-md"
+          role="button"
+        >
+          Search
+        </button>
+      </div>
+    </div>
+    <div id="products" class="mt-5">
+      <div v-if="searchResults.length === 0">
+        <h1 v-show="searchExecuted" class="black-text">No result!</h1>
+      </div>
+      <div v-else v-for="(product, index) in searchResults" :key="index" class="product mt-5 border-2 border-gray-300 p-2 rounded-md max-w-2xl">
```

Comment:
Just a general rule of thumb I don't think `v-else` (or any conditional) should be used in the same tag as a `v-for` rather wrap the for-loop in the condition.